teach_base/lesson_01/03_right_triangle.rb

```diff
@@ -0,0 +1,31 @@
+=begin
+Прямоугольный треугольник. Программа запрашивает у пользователя 3 стороны треугольника 
+и определяет, является ли треугольник прямоугольным, используя теорему Пифагора 
+и выводит результат на экран. Также, если треугольник является при этом равнобедренным 
+(т.е. у него равны любые 2 стороны), то дополнительно выводится информация о том, 
+что треугольник еще и равнобедренный.
+=end
+
+
+puts "Input three sides of your right triangle!"
+a = gets.chomp.to_f
+b = gets.chomp.to_f
+c = gets.chomp.to_f
+
+def type_triangle (a,b,c)
+  if (b < a &&  a > c) && (a**2 == c**2 + b**2)
+  	"Your triangle is right!"
+  elsif (a < b &&  b > c) && (b**2 == a**2 + c**2)
+  	"Your triangle is right!"
+  elsif (a < c &&  c > b) && (c**2 == a**2 + b**2)
+  	"Your triangle is right!"
```

Comment:
Three 'elsif's return the same result.
Try to find another way to check your conditions.

Refactor the code according to the style guide.